repository/kafka/docs/latest/security.md

```diff
@@ -31,6 +33,19 @@ $ kubectl kudo install kafka \
     -p TLS_SECRET_NAME=kafka-tls
 ```
 
+#### Using Auto certificate generation
+
+KUDO Kafka can automatically generate a CA certificate and create secrets which are then used by the brokers to set up Kafka transport encryption. 
+
+```
+kubectl kudo install kafka \
+    --instance=kafka --namespace=kudo-kafka \
+    -p TRANSPORT_ENCRYPTION_ENABLED=true \
+    -p TRANSPORT_ENCRYPTION_ALLOW_PLAINTEXT=false \
+    -p SSL_AUTHENTICATION_ENABLED=false \
+    -p USE_AUTO_TLS_CERTIFICATE=true
+```
+

```

Comment:
let's document what happens when the user provides a custom certificate and also `USE_AUTO_TLS_CERTIFICATE=true`? 
form the code it looks like it would use both auto certificate and user provided

Reply:
Updated in docs https://github.com/kudobuilder/operators/pull/201/commits/592b62012e03cdeff73c95f73a50c1361e53b206. The operator will choose Auto TLS over user provided secret